components/snackbar/snackbar.css

```diff
@@ -0,0 +1,69 @@
+.snackbar-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    gap: 1rem;
+    background-color: var(--grey);
+    padding: 1rem;
+    border-radius: 0.5rem;
+    margin-top: 1rem;
+}
+
+.snackbar-baseline {
+    max-width: 25rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding: 0.5rem;
+    background-color: var(--dark-grey);
+    color: white;
+    border-radius: 4px;
+}
+
+.retry-btn {
+    padding: 0.5rem;
+    font-weight: 600;
+    color: var(--main-color);
+}
+
+a {
+    color: var(--dark-blue);
+}
+
+.close-btn {
+    cursor: pointer;
+    padding: 0.5rem;
+    border-radius: 50%;
+}
+
+.close-btn:hover {
+    color: var(--black);
+}
+
+
+.snackbar-leading {
+    max-width: 25rem;
+    display: flex;
+    align-items: center;
+    justify-content: left;
+    padding: 0.5rem;
+    background-color: var(--dark-grey);
+    color: white;
+    border-radius: 4px;
+}
+
+.snackbar-stacked {
+    max-width: 25rem;
+    display: flex;
+    flex-direction: column;
+    padding: 0.5rem;
+    background-color: var(--dark-grey);
+    color: white;
+    border-radius: 4px;
+}
```

Comment:
you can create .snackbar as class and write the commonly used properties in all three types

Width, flex, padding, radius, color, background,

Reply:
Thankyou Nikhil for pointing it out ! I just did it :)